Library/.rubocop.yml

```diff
@@ -339,14 +339,6 @@ Style/DisableCopsWithinSourceCodeDirective:
     - "/**/{Formula,Casks}/*.rb"
     - "**/{Formula,Casks}/*.rb"
 
-# Don't enforce documentation in casks or formulae.
```

Comment:
Still need this here, I think, or to disable it or it'll trigger for taps?

Reply:
Oh, yeah. The separate files thing is confusing.